### DISCUSSION TITLE
Enable "ñ" in Spanish lazy mode

### DISCUSSION THREAD

**Discussion by DasHammett** [monkeytypegame/monkeytype#1937] [Ideas] [answered]
Hi all, 

Love lazy mode, it enables me to focus on the actual word without having to worry about the accents and the right pinky finger, but in Spanish it also disables the "ñ" character.
Since we have a dedicated key for that letter (at least in the ISO keyboard layout) it would be fantastic if the lazy mode enabled this key, so we can practice it.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I don't think that this would a viable thing to do. The whole point of lazy mode is to remove every single diacritic so that you can type faster. So it wouldn't make sense to have the "ñ" character in that it's a diacritic and would completely contradict what lazy mode is supposed to be. If you wanted to practice that letter you could type in normal spanish or create a custom word set that has that diacritic.

**Reply by SeerLite** [reply]
Too late :p #1946
If lazy mode is just to _type faster_ then ñ shouldn't be excluded, as it doesn't slow down typing in most layouts for languages that use it.
